From 3d37564726c422c567577d2263e7f10ed3d519ae Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Wed, 10 Jan 2018 15:18:53 -0800
Subject: [PATCH] TRITON-45 VOLAPI uses wrong error level when logging
 successful import of storage VM image Reviewed by: Nick Zivkovic
 <nick.zivkovic@joyent.com> Approved by: Nick Zivkovic
 <nick.zivkovic@joyent.com>

---
 lib/volapi.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/volapi.js b/lib/volapi.js
index e67abb9..f69dd1a 100644
--- a/lib/volapi.js
+++ b/lib/volapi.js
@@ -207,7 +207,7 @@ function importImage(imgapiClient, imageUuid, log, callback) {
                                 'Error when importing image');
                             importImgBackoff.backoff();
                         } else {
-                            log.error({imageUuid: imageUuid},
+                            log.info({imageUuid: imageUuid},
                                 'Imported image succesfully');
                             importImgBackoff.reset();
                             callback();
-- 
2.21.0

